server/src/service-layer/controllers/README.md

```diff
@@ -3,3 +3,9 @@
 Controller methods convert requests coming from the routes and convert to HTTP responses
 
 See the [tsoa docs](https://tsoa-community.github.io/docs/examples.html) for how to write controllers (These will automatically be converted to routes)
+
+## Stripe Webhook
```

Comment:
Great job! nice to see documentation

nit: This would be better in the wiki i think

Reply:
Moved to wiki.